src/ffi/mod.rs

```diff
@@ -1136,10 +1231,20 @@ mod tests {
     fn test_bytes_to_zero_terminated_unmanaged_bytes() {
         let inputs: [(&[u8], &[u8]); 3] = [(&[], &[0]), (&[0], &[0, 0]), (&[1, 2], &[1, 2, 0])];
         for (input, expected_output) in inputs {
-            let output = bytes_to_zero_terminated_unmanaged_bytes(&input);
+            let output = bytes_to_zero_terminated_unmanaged_bytes(input);
             let output =
                 unsafe { Vec::from_raw_parts(output as *mut u8, input.len() + 1, input.len() + 1) };
             assert_eq!(output, expected_output);
         }
     }
+
+    #[test]
+    fn test_logging_when_telio_dev_empty() {
+        let telio_dev: *mut *mut telio = ptr::null_mut();
+        let res = get_instance_from_ptr(telio_dev);
+        assert_eq!(res, None);
+        let telio_dev: *mut *mut telio = Box::into_raw(Box::new(ptr::null_mut()));
+        let res = get_instance_from_ptr(telio_dev);
+        assert_eq!(res, None);
+    }
```

Comment:
Please also add a positive case when we do get an id stored inside.

Reply:
Done